packages/config/config/devices/0x0063/ge_12730_zw4002.json

```diff
@@ -60,81 +72,6 @@
 					"value": 1
 				}
 			]
-		},
-		"7": {
-			"label": "Dim Rate Steps (Z-Wave Command)",
-			"description": "Number of steps or levels",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 99,
-			"defaultValue": 1,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"8": {
-			"label": "Dim Rate Timing (Z-Wave)",
-			"description": "Timing of steps or levels",
-			"valueSize": 2,
-			"unit": "10ms",
-			"minValue": 1,
-			"maxValue": 255,
-			"defaultValue": 3,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"9": {
-			"label": "Dim Rate Steps (Manual)",
-			"description": "Number of steps or levels",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"10": {
-			"label": "Dim Rate Timing (Manual)",
-			"description": "Timing of steps",
-			"valueSize": 2,
-			"unit": "10ms",
-			"minValue": 1,
-			"maxValue": 255,
-			"defaultValue": 3,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"11": {
-			"label": "Dim Rate Steps (All-On/All-Off)",
-			"description": "Number of steps or levels",
-			"valueSize": 1,
-			"minValue": 1,
-			"maxValue": 99,
-			"defaultValue": 1,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
-		},
-		"12": {
-			"label": "Dim Rate Timing (All-On/All-Off)",
-			"description": "Timing of steps or levels",
-			"valueSize": 2,
-			"unit": "10ms",
-			"minValue": 1,
-			"maxValue": 255,
-			"defaultValue": 3,
-			"unsigned": true,
-			"readOnly": false,
-			"writeOnly": false,
-			"allowManualEntry": true
 		}
 	},
 	"compat": {
```

Comment:
I guess it doesn't hurt, but the Supervision CC class removal is not necessary for this device. It's Z-Wave Classic, so no Supervision CC, and it's therefore not a Fan Switch Device Type. The removal is only required for the GE Z-Wave Plus devices that report as Fan Switch DT and don't support Supervision CC.

The treatBasicSetAsEvent is apparently unnecessary as well since it looks like double taps are not supported, due to no association groups?

Reply:
It is necessary. The zw4002 hardware was breaking with the 14287 and this device due to supervision. This fixes a prior issue.

Reply:
The 12730 is not a 14287. `ge_12730_zw4002.json `.